test/e2e/sched/sched_test.go

```diff
@@ -174,5 +181,60 @@ var _ = Describe("[Scheduler] imageReplacement", func() {
 				return true
 			}).WithTimeout(time.Minute * 2).WithPolling(time.Second * 30).Should(BeTrue())
 		})
+		It("should reflect changes in cacheResyncPeriod when configured", func() {
+			nroSchedObj := objects.TestNROScheduler()
+			Expect(e2eclient.Client.Get(context.TODO(), client.ObjectKeyFromObject(nroSchedObj), nroSchedObj)).ToNot(HaveOccurred())
+
+			t := time.Minute * 5
+			nroSchedObj.Spec.CacheResyncPeriod = &metav1.Duration{Duration: t}
+			Expect(e2eclient.Client.Update(context.TODO(), nroSchedObj)).ToNot(HaveOccurred())
+
+			By("checking cacheResyncPeriod under the CR's Status")
```

Comment:
I'm not sure if it triggers an update on the pods though, in case you don't have an env to check it let me know and I'll verify that

Reply:
this setting will affect the scheduler plugin pod not the RTEs though. The general concept however is still relevant.

Reply:
yes. we'll wait. i'll add `Eventually` as Francesco suggested.

Reply:
I think this block belongs in a `Eventually`. The `Update` can fail because the `resourceVersion` doesn't match, but if the resourceVersion doesn't match the only possible fix is `Get` again.